### PR TITLE
Added test file predict listing price

### PR DESCRIPTION
**Pull request by spencermendelsohn**
Branch to talk to endpoint for our model

### CODE REVIEW

File: tests/test-deployment.py
Comment (owens1127) [inline tests/test-deployment.py:1]: **Major**: This should not be in the test folder

**Review by owens1127 — CHANGES_REQUESTED** [review]
changes requested

**Review by mlebiedzinski — APPROVED** [review]
epic

File: src/predict-price.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    print("Body content:")
+    print(body.decode("utf-8"))
+
+    response = sagemaker_runtime.invoke_endpoint(
+        EndpointName=endpoint,
+        ContentType="text/csv",
+        Body=body,
+        Accept="application/json",
+    )
+    output = json.loads(response["Body"].read().decode("utf-8"))
```
Comment (owens1127) [inline src/predict-price.py]: We should use a DTO to send the data to our frontend as to not couple the AWS response to our frontend

**Review by owens1127 — APPROVED** [review]
Thanks for the changes, LGTM

**Pull request merged**